post-processing/fuzz_analysis.py

```diff
@@ -378,8 +377,9 @@ def detect_branch_level_blockers(
             continue
 
         llvm_branch_profile = functions_profile[function_name].branch_profiles
+        source_file_path = functions_profile[function_name].function_source_file
         # Just extract the file name and skip the path
-        source_file_name = functions_profile[function_name].function_source_file.split('/')[-1]
+        source_file_name = source_file_path.split('/')[-1]
```

Comment:
Just use `os.path.basename` here?

Reply:
These are populated from debug info in an LLVM pass, so I think does not make sense to use os.path etc.

Reply:
Could you elaborate a bit more? os.path.basename works generally any any path strings. 

It is however, tied to the OS it's running on. I.e. If this script runs on Windows on something generated from Linux, it might not work.

Reply:
My bad! I misunderstood your initial comment. Using `os.path.basename` now.